source/IncomingMessages.Domain/MessageParsers/RSM015/RequestValidatedMeasurementsJsonMessageParser.cs

```
@@ -0,0 +1,76 @@
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Text.Json;
+using Energinet.DataHub.EDI.BuildingBlocks.Domain.Models;
+using Energinet.DataHub.EDI.IncomingMessages.Domain.Messages;
+using Energinet.DataHub.EDI.IncomingMessages.Domain.Schemas.Cim.Json;
+using Microsoft.Extensions.Logging;
+
+namespace Energinet.DataHub.EDI.IncomingMessages.Domain.MessageParsers.RSM015;
+
+public class RequestValidatedMeasurementsJsonMessageParser(
+    JsonSchemaProvider schemaProvider,
+    ILogger<RequestValidatedMeasurementsJsonMessageParser> logger)
+    : JsonMessageParserBase(schemaProvider, logger)
+{
+    private const string ValueElementName = "value";
+    private const string MridElementName = "mRID";
+    private const string EndElementName = "end_DateAndOrTime.dateTime";
+    private const string StartElementName = "start_DateAndOrTime.dateTime";
+    private const string MeteringPointIdentificationElementName = "marketEvaluationPoint.mRID";
+
+    public override IncomingDocumentType DocumentType => IncomingDocumentType.RequestValidatedMeasurements;
+
+    public override DocumentFormat DocumentFormat => DocumentFormat.Json;
+
+    protected override string HeaderElementName => "RequestValidatedMeasureData_MarketDocument";
+
+    protected override string DocumentName => "RequestValidatedMeasureData";
+
+    protected override IIncomingMessageSeries ParseTransaction(JsonElement transactionElement, string senderNumber)
+    {
+        var id = transactionElement.GetProperty(MridElementName).GetString() ?? string.Empty;
```

We should throw an exception if the id does not exist